cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -42,6 +42,8 @@ describe('CourseOutlinePage', function() {
             user_partition_info: {},
             highlights_enabled: true,
             highlights_enabled_for_messaging: false,
+            hide_from_toc: false,
```

Comment:
I remember tests failing because of this being undefined: https://github.com/openedx/edx-platform/pull/33952/files#diff-5543f4f9dab9a8708be7495f014ee8a77c2e5f75b71dd69b4c921babc49c86a4R9. I'll test in more detail later.

Reply:
This is not a test, it's a mock definition for subsections: https://github.com/openedx/edx-platform/pull/33952/files#diff-8fade4a4fb620b73515c7e83dad5d3af25f1b301c68464aadf8846bff93fbe26R2236-R2249

If not added to sections, subsections, and unit mocks, it won't have a default as when testing in a real environment, where defaults come from the backend.

So, without those changes, tests fail because both enableHideFromTOCUI and hideFromTOC are undefined. So I'll leave it as is. Let me know if you have any objections.

Reply:
No objections. Thanks for the explanation